system_test/aest_hard_fork_SUITE.erl

```diff
@@ -638,9 +638,10 @@ restore_db_backup_on_node(NodeName, TarBin, Content, DestDir, Cfg) ->
     ct:log("Restoring DB backup of byte size ~p on node ~s",
            [byte_size(TarBin), NodeName]),
     _ = aest_nodes:run_cmd_in_node_dir(NodeName, ["mkdir", DestDir], Cfg),
-    "" = aest_nodes:run_cmd_in_node_dir(NodeName, ["ls", DestDir], Cfg),
+    {_, ""} = aest_nodes:run_cmd_in_node_dir(NodeName, ["ls", DestDir], Cfg),
```

Comment:
These should probably match on the exit code as well? Same for the calls to `run_cmd_in_node_dir` below.

Reply:
There wasn't any check of the status code before, I just kept the exact same behavior.